backend/de.metas.business/src/main/java/de/metas/invoice/callout/C_Invoice.java

```diff
@@ -182,4 +186,17 @@ public void updateBPartnerAddressForceUpdateCapturedLocation(final I_C_Invoice i
 		documentLocationBL.updateCapturedLocation(InvoiceDocumentLocationAdapterFactory.locationAdapter(invoice));
 
 	}
+
+	@CalloutMethod(columnNames = { I_C_Invoice.COLUMNNAME_C_PaymentTerm_ID, I_C_Invoice.COLUMNNAME_DateInvoiced }, skipIfCopying = true)
```

Comment:
is it OK to do this on DateInvoiced, too?
i checked PaymentTerm.computeDueDate and it seems that the iver date would be overridden, even if `IsAllowOverrideDueDate` is `true`

Reply:
ok, i guess the DueDate is never set via UI-Field-change..then i think it might be better to remove the dateInvoiced from the `@CalloutMethod` anotation, in order to reduce FUD

Reply:
I think in this case, we're not really talking about overriding the `DueDate`, but setting the "initial" value for it (based on PaymentTerm and DateInvoiced). So, if we change the `DateInvoiced`, we'll set the initial (calculated) value of the `DueDate` for that `DateInvoiced`, even though, the PaymentTerm has `IsAllowOverrideDueDate` = `false`.
Then, if we really want to override the initial value, we will use the process and verify that the payment term allows it.